src/main/java/beam/agentsim/events/ReplanningEvent.java

```diff
@@ -0,0 +1,50 @@
+package beam.agentsim.events;
+
+import org.matsim.api.core.v01.Id;
+import org.matsim.api.core.v01.events.Event;
+import org.matsim.api.core.v01.population.Person;
+import org.matsim.core.api.internal.HasPersonId;
+
+import java.util.Map;
+
+public class ReplanningEvent extends Event implements HasPersonId {
+
+    public final static String EVENT_TYPE = "Replanning";
```

Comment:
⚠️ This class already have a method exposing this value (`getEventType`). Consider making the constant private and avoiding redundance